### DISCUSSION TITLE
Attribution Model Question Regarding Emissions

### DISCUSSION THREAD

**Discussion by ghost** [USEPA/supply-chain-factors#67] [Q&A] [answered]
If we had specific questions regarding the GHG Attribution Models could you point us to the best source to ask our questions?

- Our question is regarding emissions for the NAICS Code - 515200: Cable and other subscription programming. Why are HFCs emissions in this NAICS code significantly higher than other related industries? _(i.e., 512100: Motion picture and video industries, 515100: Radio and television broadcasting, 517100: Wired telecommunications carriers, etc.)_

**Comment by bl-young** [comment, marked as answer]
This discussion board is likely your best option for questions.

With the caveat that identifying specific differences between sectors can be tricky and time-consuming:

The HFC emissions you describe likely stem from Commercial Refrigeration and Commercial Air Conditioning, which can be found in the method [here](https://github.com/USEPA/flowsa/blob/v1.3.1/flowsa/methods/flowbysectormethods/GHG_national_m1.yaml#L626-L662), from GHGI Table 4-102 (see below). Those emissions are attributed to sector based on purchase of air conditioning and refrigeration equipment (`333415`). As such, the sector Cable and other subscription programming likely makes more purchases of that equipment than the other sectors you mention, resulting in higher attributed emissions of that flow.

![image](https://github.com/USEPA/supply-chain-factors/assets/44471635/bf882ce1-967f-4cfa-bd23-666601cbc2bd)
